File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.c
#include <stdio.h>
int main(void)
{
    int n, cnt=0, num=666;
    scanf("%d", &n);
    while(1)
    {
        int tmp=num;
        while(tmp!=0)    //숫자를 일의 자리부터 때어내가며 666이 있는지 확인하는 반복
        {
            if(tmp%1000==666){
                cnt++;
                break;
            }
            else
                tmp/=10;
        }
        if(cnt==n){
            printf("%d\n", num);
            break;
        }
        num++;
    }
    return 0;
}

File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.py
n = int(input())
cnt, num = 0, 666
while True:
    tmp = num
    while tmp:
        if tmp % 1000 == 666:
            cnt += 1
            break
        else:
            tmp //= 10
    if cnt == n:
        print(num)
        break
    num += 1